etc/openqa/openqa.ini

```diff
@@ -58,6 +58,9 @@
 # this domain, this job is labeled as linked and considered as important
 # recognized_referers = bugzilla.suse.com bugzilla.opensuse.org bugzilla.novell.com bugzilla.microfocus.com progress.opensuse.org github.com
 
+## A regex in a string of test settings to ignore for "job investigation"
+#job_investigate_ignore = "(JOBTOKEN|NAME)"
```

Comment:
Since the tab is called "Investigation" perhaps so should the setting? And, would it be worth introducing a group in case we want more settings?

Reply:
yeah well, it's used in https://github.com/os-autoinst/openQA/pull/2609/files#diff-85ae48e70a5c110c9e439c3a5ea28d5fR1837 within the method "investigate" so both would be possible :) Currently I have no idea for other settings so I would not do any over-engineering unless we need it. I don't forsee problems with moving this setting to a different section. Should be fine.